src/lib.rs

```diff
@@ -23,9 +23,18 @@ pub use ac_compose_macros::*;
 pub use ac_node_api::*;
 pub use ac_primitives::*;
 pub use api::*;
-pub use utils::*;
 
 pub mod api;
 pub mod extrinsic;
 pub mod rpc;
-pub mod utils;
```

Comment:
I'm wondering if the tag `E1-breaksnothing` is accurate given that we remove the `utils` module. Couldn't this affect some users?

Reply:
Agreed, it does break the api. Will correct this

Reply:
Thanks for pointing it out!